ichat.react/src/sidebar/Sidebar.js

```diff
@@ -98,16 +101,49 @@ class Sidebar extends React.Component {
     this.fetchConversations();
   }
 
+  updatePageTitle() {
+    var unreadMessageText = "";
+    this.setState({unreadMessage:this.getUnreadMessageCount()});
+    if(this.state.unreadMessage > 0 &&this.state.unreadMessage != null)
```

Comment:
unreadMessage doesn't need to be in the state. Can use a local variable instead.